kano_settings/boot_config.py

```diff
@@ -104,14 +104,22 @@ def check_corrupt(self):
         except:
             return True
 
-        must_contain = set(['overscan_bottom', 'over_voltage'])
+        must_contain = set(['dtparam'])
         found = set()
 
         for l in lines:
             for m in must_contain:
                 if m in l:
                     found.add(m)
-        return must_contain != found
+
+        if must_contain == found:
+            return False
```

Comment:
The implementation here is fundamentally flawed; users should be free to change anything in the config.txt file without our libs breaking / resetting. I suggest the function is removed completely.

Reply:
Right now this is the only way that we know if the file has become corrupted. I'm happy for you to create ticket to remove this but at this point I'm not changing the fundamental implementation to allow for users to arbitrarily modify the `config.txt` (which we randomly replace anyway)

Reply:
Just for to keep in mind for later, I already implemented a mechanism to achieve this limitation, explained here:

 * https://github.com/KanoComputing/kano-settings/#display

and implemented here:

 * https://github.com/KanoComputing/kano-settings/blob/master/etc/init.d/kano-settings#L27